lightspeed/modules/proc_monitor-onpremise-deployment.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: PROCEDURE
+
+[id="monitor-lightspeed-onpremise-deployment_{context}"]
+
+= Monitoring {LightspeedShortName} on-premise deployments
+
+After the {LightspeedShortName} on-premise deployment is successful, use the following procedure to monitor the metrics on an API endpoint `/api/v2/metrics`. 
+
+.Prerequisites
+* You have set up a {LightspeedShortName} on-premise deployment.
+
+.Procedure
+
+. Create a *system auditor* user:
+.. Create a user in the {PlatformName} with *system auditor* role.
++
+To know the procedure to create a user with *system auditor* role, see the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-creating-a-user[Creating a user] in the _Automation Controller User Guide_.
```

Comment:
I wonder if this statement could be rolled into the step? For example,

Use the steps for link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-creating-a-user[Creating a user] to create a user in the {PlatformName} with the system auditor role. 

That way the step includes the link to the procedure rather than the user having to look for it after the fact? Your choice.

Reply:
Updated